src/main/java/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,115 @@
+package dao;
+
+import exceptions.DataProcessingException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import model.Manufacturer;
+import util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertRequest = "INSERT INTO manufacturers(name, country) values(?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createStatement =
```

Comment:
```suggestion
        String query = "INSERT INTO manufacturers(name, country) values(?,?);";
        try (Connection connection = ConnectionUtil.getConnection();
                 PreparedStatement statement =
```
you can use simpler names for queries and statements because it's obvious what they are from the context